erc/ERCS/erc-4337.md

```diff
@@ -379,9 +379,11 @@ The simulated call performs the full validation, by calling:
 2. `account.validateUserOp`.
 3. if specified a paymaster: `paymaster.validatePaymasterUserOp`.
 
-Either `validateUserOp` or `validatePaymasterUserOp` may return a "validAfter" and "validUntil" timestamps, which is the time-range that this UserOperation is valid on-chain.
-The simulateValidation call returns this range.
-A node MAY drop a UserOperation if it expires too soon (e.g. wouldn't make it to the next block)
+The simulateValidation should validate the return value (validationData) returned by the account's `validateUserOp` and paymaster's `validatePaymasterUserOp`.
```

Comment:
You describe implementation details of the EntryPointSimulations. It seems out of place